picard/ui/itemviews/columns.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2006-2008, 2011-2012 Lukáš Lalinský
+# Copyright (C) 2007 Robert Kaye
+# Copyright (C) 2008 Gary van der Merwe
+# Copyright (C) 2008 Hendrik van Antwerpen
+# Copyright (C) 2008-2011, 2014-2015, 2018-2024 Philipp Wolfer
+# Copyright (C) 2009 Carlin Mangar
+# Copyright (C) 2009 Nikolai Prokoschenko
+# Copyright (C) 2011 Tim Blechmann
+# Copyright (C) 2011-2012 Chad Wilson
+# Copyright (C) 2011-2013 Michael Wiencek
+# Copyright (C) 2012 Your Name
+# Copyright (C) 2012-2013 Wieland Hoffmann
+# Copyright (C) 2013-2014, 2016, 2018-2024 Laurent Monin
+# Copyright (C) 2013-2014, 2017, 2020 Sophist-UK
+# Copyright (C) 2016 Rahul Raturi
+# Copyright (C) 2016 Simon Legner
+# Copyright (C) 2016 Suhas
+# Copyright (C) 2016-2017 Sambhav Kothari
+# Copyright (C) 2018 Vishal Choudhary
+# Copyright (C) 2020-2021 Gabriel Ferreira
+# Copyright (C) 2021 Bob Swift
+# Copyright (C) 2021 Louis Sautier
+# Copyright (C) 2021 Petit Minion
+# Copyright (C) 2023 certuna
+# Copyright (C) 2024 Suryansh Shakya
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+
+from collections.abc import MutableSequence
+from enum import IntEnum
+
+from PyQt6 import QtCore
+
+from picard.i18n import N_
+from picard.util import icontheme
+
+
+class ColumnAlign(IntEnum):
+    LEFT = 0
+    RIGHT = 1
+
+    def __repr__(self):
+        cls_name = self.__class__.__name__
+        return f'{cls_name}.{self.name}'
+
+
+class ColumnSortType(IntEnum):
+    NONE = 0
+    NAT = 1
+    DEFAULT = 2
```

Comment:
Just a minor thing, but maybe either move DEFAULT to index 1, because NAT is the more special case and/or rename it from default to TEXT, because it is just normal alphabetical text order

Reply:
Done in last iteration, rebased.
I also added a check for sortkey when sort_type is SORTKEY